### PR TITLE
Add git tag trigger to container build wf

### PR DESCRIPTION
**Pull request by mquhuy**

### CODE REVIEW

File: .github/workflows/build-images-action.yml
Comment (tuminoid) [inline .github/workflows/build-images-action.yml:1]: L18, typo `mariabd`.

File: .github/workflows/build-images-action.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - 'main'
       - 'release-*'
+    tags:
+      - 'capm3-v*'
```
Comment (tuminoid) [inline .github/workflows/build-images-action.yml:8]: More inconsistent quoting? Most tags triggers in other repos did not have quotes.

Comment (mquhuy) [reply .github/workflows/build-images-action.yml:8]: As I explained in another PR, this is related to the dash character.

**Review by Rozzii — APPROVED** [review]
I approve but I have no rights in this repo.

**Comment by tuminoid** [comment]
LGTM as well.

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
I've removed mariadb build trigger from Quay, this will be the image trigger from now on.

**Pull request merged**